README.md

```diff
@@ -421,4 +422,52 @@ _Matt Ricketson, Taylor Kelly_
    - `onDrag {}` and `onDrop {}` to support drag & drop
    - `UniformTypeIdentifers` framework for e.g. introspection of file types
    - `Sign in with Apple` as first class component in SwiftUI • available on every platform
-   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
\ No newline at end of file
+   - Other Frameworks: **AuthenticationServices**, **AVKit**, **MapKit**, **SceneKit**, **SpriteKit**
+
+## Configure and link your App Clips
+
+https://developer.apple.com/videos/play/wwdc2020/10146/
```

Comment:
If possible please use the short url version, it is easier to parse later on for me :-) You can just copy & paste one of the others and change the session number.